.github/workflows/check-file-updates.yaml

```diff
@@ -0,0 +1,43 @@
+name: Check config and readme updates
+on:
+  pull_request:
+jobs:
+  file-updates:
+    permissions:
+      pull-requests: write
+    name: Ensure generated files are included
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Generate files
+        id: generate-files
+        run: |
+          CMD="make generate manifests api-docs"
+          $CMD
+          echo "CMD=$CMD" >> $GITHUB_OUTPUT
```

Comment:
This is how I originally started, but wanted to be more fancy with `fc` :) that's fine I guess, though shellcheck will freak out :)